packages/builder/src/pages/builder/portal/account/auditLogs/index.svelte

```diff
@@ -86,8 +86,15 @@
   $: userPage = $userPageInfo.page
   $: logsPage = $logsPageInfo.page
 
+  $: userList = {
+    ...(userList || {}),
+    ...$users.data?.reduce((p, c) => {
```

Comment:
What are letters 'p' and 'c' in this case? Maybe give them full variable names?

Reply:
`previous` and `current`. It's the way the js reduce works

Reply:
I renamed it with the full variable names